packages/core/src/app/shipping/ShippingAddress.tsx

```diff
@@ -24,6 +24,7 @@ export interface ShippingAddressProps {
     countriesWithAutocomplete: string[];
     formFields: FormField[];
     googleMapsApiKey?: string;
+    isGuest?: boolean;
```

Comment:
Based on my prev comment, I don't think `isGuest` should be provided to shipping components

Reply:
removed

Reply:
removed